warp/codegen.py

```diff
@@ -1771,6 +1771,22 @@ def emit_If(adj, node):
                 out = adj.add_builtin_call("where", [cond, var1, var2])
                 adj.symbols[sym] = out
 
+    def emit_IfExp(adj, node):
+        cond = adj.eval(node.test)
+
+        if cond.constant is not None:
+            return adj.eval(node.body) if cond.constant else adj.eval(node.orelse)
```

Comment:
Just for good measure, could you please add a couple of tests to cover the constant case such as `x if 123 else y` and `x if False else y`?